### PR TITLE
doc/website: new design

### PR DESCRIPTION
**Pull request by elad-n**

### CODE REVIEW

**Comment by a8m** [comment]
Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic. 

Header links are broken.
image

Footer links are too big and the indentation is weird on small screens. 
image
image

**Comment by a8m** [comment]
Also, the logo is 44K in size. Please, move it out from this repo to CDN (cc @rotemtam).

File: doc/website/docusaurus.config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   baseUrl: '/',
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
+  favicon: 'img/logoAtlas.svg',
```
Comment (a8m) [inline doc/website/docusaurus.config.js:9]: 45KB size image for the favicon?

**Comment by a8m** [comment]
Why both logoAtlas.svg and logo.svg are needed? It's the same file

File: doc/website/docusaurus.config.js (hunk 1 above)
Comment (elad-n) [reply doc/website/docusaurus.config.js]: replaced to a ico file.

**Comment by elad-n** [comment]
> Why both logoAtlas.svg and logo.svg are needed? It's the same file

ya, the first one should have been an ico format. i replaced it. so now its ok.
now we have one for the favicon and one for the navbar  image

**Comment by elad-n** [comment]
> Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic.
> 
> Header links are broken. image
> 
> Footer links are too big and the indentation is weird on small screens. image image



> Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic.
> 
> Header links are broken. image
> 
> Footer links are too big and the indentation is weird on small screens. image image

The header link won't
Screen Shot 2022-01-13 at 22 43 57
Screen Shot 2022-01-13 at 22 44 05
 work on localhost no ? 
the rest is fixed.

**Comment by rotemtam** [comment]
Thanks for working on this @elad-n !

Some feedback. 

Before the "hero" image loads, site looks like:
Screen Shot 2022-01-15 at 12 13 09
Let's pick a bg color more similar to the image so it doesn't look weird.
---
Screen Shot 2022-01-15 at 12 14 37

H1 is centered.. this looks odd, can we revert it to the way it normally looks? 

---

Screen Shot 2022-01-15 at 12 16 02
Navbar, should look like the design.  Spacing from top to text/icons is off, and text should be aligned to the right (see how it is in the design). 

Footer, discord link takes me to 404.

File: doc/website/src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                </Link></button>
+
+                <img src="https://atlasgo.io/uploads/landing/tables.png" alt="ent"/>
+               <section className={index.section} style={{ marginTop: "10px" }}>
+                   <h2 className={index.title2} style={{ textAlign: "center", color: "#000939" }}>
+                       Visualize your schemas <br/> using the <span style={{color: "#2064E9"}}>Atlas UI</span>
+                   </h2>
+                   <p className={index.paragraph2}
+                      style={{ color: "#757889" }}>
+                       With a simple command line flag, Atlas opens up as a rich user interface to visualize your data topology and manage your data topology.
```
Comment (rotemtam) [inline doc/website/src/pages/index.js:100]:
```suggestion
                       With a single command-line flag, Atlas opens up as a rich user interface to visualize and manage your data topology.
```

File: doc/website/docusaurus.config.js [hunk 3]
```diff
@@ -19,68 +19,44 @@ module.exports = {
...
-            {
-              label: 'Welcome',
-              to: '/',
-            },
-          ],
+          "title": "Docs",
+          "items": [
+            {"label": "Getting Started", "to": "cli/getting-started/setting-up"},
+            {"label": "Data Definition Language ", "to": "ddl/intro"},
+            {"label": "CLI Reference", "to": "CLI/atlas"},
```
Comment (a8m) [inline doc/website/docusaurus.config.js:41]:
```suggestion
            {"label": "CLI Reference", "to": "cli/atlas"},
```

why the URL is uppercased?

**Review by a8m — APPROVED** [review]

**Review by rotemtam — APPROVED** [review]

**Pull request merged**